conda/plugins/__init__.py

```diff
@@ -8,3 +32,6 @@
     CondaSubcommand,
     CondaVirtualPackage,
 )
+
+# Used for sphinx-autoapi
+__all__ = ()
```

Comment:
Remember that this will ALSO prevent imports such as `from conda.plugins import *` from working

Reply:
@jezdez, I removed these for now.